### ISSUE TITLE
Batch normalization missing

### ISSUE THREAD

**Issue by ArlindKadra** [automl/Auto-PyTorch#297]
I noticed that batch normalization is missing in the `MLP` and `ShapedMLP` backbone, we should probably add it to both implementations.
